Src/BootCamp.Chapter/lesson3.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class lesson3
```

Comment:
Class and file name should be written with a capital L.

Reply:
I corrected it earlier, could you check on recent commits?